### ISSUE TITLE
Extract-AppFileToFolder misplaces rdlc layout files

### ISSUE THREAD

**Issue by ChrisBlankDe** [microsoft/navcontainerhelper#1992]
**Describe the issue**
`Extract-AppFileToFolder` misplaces rdlc layout files and leaves Sources Code uncompilable (be sure to set generate report layout to No).

Tree In Extracted App:
```
layout
  src
    Layout
      KFW Master Document.rdlc
src
  src
    report
      ArchivedSalesQuote.Report.al (RDLCLayout = 'src/Layout/KFW Master Document.rdlc';)
```
Tree In App:
```
src
  Layout
    KFW Master Document.rdlc
  report
    ArchivedSalesQuote.Report.al (RDLCLayout = 'src/Layout/KFW Master Document.rdlc';)
```

 Error while compiling:

> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\ProgramData\BcContainerHelper\019ee0df-f1b6-4b6d-9259-4d00735be914\src\Layout\KFW Master Document.rdlc'.

**Scripts used to create container and cause the issue**
```
Extract-AppFileToFolder -appFilename "C:\Users\***\Downloads\Kumavision AG_KUMAVISION base_18.2.11364.0.app" -appFolder C:\ProgramData\BcContainerHelper\019ee0df-f1b6-4b6d-9259-4d00735be914 -generateAppJson
Compile-AppInBcContainer -containerName appsource -appProjectFolder C:\ProgramData\BcContainerHelper\019ee0df-f1b6-4b6d-9259-4d00735be914 -GenerateReportLayout No -credential $Credential
```

**Full output of scripts**
```
Extracting C:\Users\***\Downloads\Kumavision AG_KUMAVISION base_18.2.11364.0.app
...
C:\ProgramData\BcContainerHelper\019ee0df-f1b6-4b6d-9259-4d00735be914\src\src\tableextension\PurchRcptLine.TableExt.al(211,21): warning AS0079: The procedure 'OnBeforeShowEditPurchRcptLine' in TableExtension 'KVSKBAPurchRcptLine' must hav
e at least one of the mandatory affixes 'KVS' to prevent breaking dependent extensions that reference it in case a procedure with the same signature is introduced by the base object or by another extension.

Compilation ended at '17:45:14.218'.

System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\ProgramData\BcContainerHelper\019ee0df-f1b6-4b6d-9259-4d00735be914\src\Layout\KFW Master Document.rdlc'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.IO.File.ReadAllBytes(String path)
   at Microsoft.Dynamics.Nav.CodeAnalysis.RelativeFileSystem.ReadBytes(String path) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\Compilation\RelativeFileSystem.cs:line 23
   at Microsoft.Dynamics.Nav.CodeAnalysis.Compilation.WriteReportLayout(ISymbol reportOrExtensionSymbol, ModuleOutputter outputter) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\Compilation\Compilation.cs:line 984
   at Microsoft.Dynamics.Nav.CodeAnalysis.Compilation.WriteToOutput(ModuleOutputter outputter, ModuleBuilder moduleBuilder, Stream xmlDocComments, DiagnosticBag diagnostics) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\Comp
ilation\Compilation.cs:line 945
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.CommonCompiler.PackageTheModule(Compilation compilation, CompilerTextWriter consoleOutput, CancellationToken cancellationToken, ErrorLogger errorLogger, AnalyzerDriver analyzerDriver, 
Predicate`1 filterOpt, Predicate`1 radCompilationFilter) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\CommandLine\CommonCompiler.cs:line 749
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.CommonCompiler.RunCoreImpl(CompilerTextWriter consoleOutput, ErrorLogger errorLogger, Compilation compilation, AnalyzerDriver analyzerDriver, IEnumerable`1 analyzerExceptionDiagnostics
, IEnumerable`1 additionalTextFiles, CancellationToken cancellationToken) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\CommandLine\CommonCompiler.cs:line 526
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.CommonCompiler.RunCore(CompilerTextWriter consoleOutput, ErrorLogger errorLogger, CancellationToken cancellationToken) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\Comma
ndLine\CommonCompiler.cs:line 473
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.CommonCompiler.Run(CompilerTextWriter consoleOutput, CancellationToken cancellationToken) in D:\a\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\CommandLine\CommonCompiler.cs:line
 331
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.Alc.<>c__DisplayClass1_0.<Run>b__0(CompilerTextWriter textWriterOut, CompilerTextWriter _) in D:\a\1\s\source\Prod\alc\Alc.cs:line 37
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.ConsoleUtil.RunWithOutput[T](Boolean useUtf8Encoding, Func`3 func) in D:\a\1\s\source\Prod\alc\ConsoleUtil.cs:line 25
   at Microsoft.Dynamics.Nav.CodeAnalysis.CommandLine.Alc.Run(String clientDirectory, String baseDirectory, String[] args, IAnalyzerAssemblyLoader analyzerLoader) in D:\a\1\s\source\Prod\alc\Alc.cs:line 37
App generation failed with exit code -1
```

**Additional context**
- does it happen all the time? Yes
- did it use to work? Don't know.

**Comment by freddydk** [comment]
Any chance you could email me the app to freddyk at microsoft dot com, then I have a very easy repro.
Thanks

**Comment by ChrisBlankDe** [comment]
You just got a new email. :)

**Comment by freddydk** [comment]
A new preview containerhelper should be available later today.
you can also grab the fix from the checkin and apply manually

**Comment by freddydk** [comment]
Shipped in latest prerelease

**Comment by ChrisBlankDe** [comment]
When will the prerelease be  available on psgallery?

**Comment by freddydk** [comment]
Now, sorry about the delay

**Comment by freddydk** [comment]
Shipped in 2.0.15
